auth/keycloak/config/realm-export.json

```diff
@@ -878,6 +878,38 @@
         "account": ["view-profile", "manage-account"]
       },
       "notBefore": 0,
+      "groups": ["/Real Estate Manager", "/SRES"]
+    },
+    {
+      "id": "6c384423-c113-46c5-b23a-14680f4e7b78",
+      "createdTimestamp": 1600787542788,
+      "username": "sresfm",
```

Comment:
A **SRES Financial Manager** can view the **SPL Reports** and they can edit projects after they are **Disposed**